src/components/DatasetsAndDocuments/index.stories.js

```diff
@@ -0,0 +1,22 @@
+import React from "react";
+
+import DatasetsAndDocuments from ".";
+
+import { datasetTypeArgs, documentsArg } from "@/pesayetu/config";
+
+const items = [
+  { label: "DOCUMENTS & SPEECHES", children: documentsArg },
+  { label: "DATASET", children: datasetTypeArgs },
```

Comment:
These don't come from the CMS?

Reply:
@kilemensi This should be in another component